meshtastic/tests/test_slog_power_logger.py

```
@@ -139,6 +160,40 @@ def test_root_dir_legacy_alias_warns_once(
     assert "root_dir()" in str(deprecations[0].message)
 
 
+@pytest.mark.unit
+def test_slog_directory_collision_error_includes_path_and_attempts() -> None:
+    """SlogDirectoryCollisionError should include app dir and retry count context."""
+    error = slog_module.SlogDirectoryCollisionError("/tmp/slog-root", 7)
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/slog-root"

(S108)